src/includes/getting-started-config/android.mdx

```diff
@@ -0,0 +1,9 @@
+Configuration is done via `AndroidManifest.xml`:
+
+```xml {filename:AndroidManifest.xml}
+<application>
+  <meta-data android:name="io.sentry.dsn" android:value="___PUBLIC_DSN___" />
+  <meta-data android:name="io.sentry.session-tracking.enable" android:value="true" />
+  <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="30000" />
```

Comment:
I'd omit `io.sentry.session-tracking.timeout-interval-millis`, the default value is already good.